Source/WebKit/GPUProcess/media/RemoteVideoFrameObjectHeap.cpp

```diff
@@ -56,7 +56,8 @@ static WorkQueue& remoteVideoFrameObjectHeapQueue()
 Ref<RemoteVideoFrameObjectHeap> RemoteVideoFrameObjectHeap::create(Ref<IPC::Connection>&& connection)
 {
     auto heap = adoptRef(*new RemoteVideoFrameObjectHeap(WTFMove(connection)));
-    heap->m_connection->addWorkQueueMessageReceiver(Messages::RemoteVideoFrameObjectHeap::messageReceiverName(), remoteVideoFrameObjectHeapQueue(), heap);
+    Ref workQueue { remoteVideoFrameObjectHeapQueue() };
+    heap->m_connection->addWorkQueueMessageReceiver(Messages::RemoteVideoFrameObjectHeap::messageReceiverName(), workQueue, heap);
```

Comment:
Check with @rniwa but I think maybe we just need to add `Singleton` to this function name to silence the warning:
```
remoteVideoFrameObjectHeapQueue()
```

Reply:
Yeah, we can rename `remoteVideoFrameObjectHeapQueue` to `remoteVideoFrameObjectHeapQueueSingleton` and the warning should go away.

Reply:
I confirmed with him--fixed!